src/main/python/image_processing/color_manager.py

```diff
@@ -7,12 +7,36 @@ class ColorManager:
     The Color manager is responsible for applying several color management techniques
     to image passed.
     """
-    def __init__(self):
+    def __init__(self, type, channel="green", kernel_size=(13, 7)):
         """
         Initialise the Colour Manager
         """
+        self.type = type
+        self.channel = channel
+        self.kernel_size = kernel_size
+
         print("Initialise Color Manager")
 
+    def apply(self, image):
+        """
+        This performs the specified processing technique
+        Author(s):
+            Nicolai van Niekerk
+        Args:
+            image: The image to which the technique must be applied
+        Returns:
+            obj:'OpenCV image': The modified image
+        Todo:
+        """
+        if self.type == "histogram":
+            return self.histEqualisation(image)
+        elif self.type == "extract":
+            return self.extractChannel(image, self.channel)
+        elif self.type == "blackHat":
+            return self.blackHat(image, self.kernel_size)
+        elif self.type == "topHat":
```

Comment:
Instead of having only `elif` statements, add an `else` clause to use the default value for in case someone makes a spelling mistake somewhere. Also add a print statement to show which type of colour manager we are using so that we can see it in the server.